### DISCUSSION TITLE
How to turn off colored dots next to workspace numbers?

### DISCUSSION THREAD

**Discussion by pjz** [regolith-linux/regolith-desktop#702] [Q&A] [answered]
How does one turn off the colored dots next to the workspace numbers?  I can't find anything in the i3 docs.

**Comment by pjz** [comment]
Minor update: It looks like something is setting the workspace name to, like '3;3 •' or something (can't really cut-and-paste the workspace name to be exact).

Also: there's no way to set 'strip_workspacename' in the regolith bar config without editing `/usr/share/regolith/i3/config.d/70_bar`.

**Comment by pjz** [comment, marked as answer]
Found the answer on slack: Just define all your `i3-wm.workspace.01.name:` with something like  `1:<span>1</span>` because it's defined in `/usr/share/regolith-look/default/i3xrocks` where it does `#define typeface_bar_glyph_workspace` - not easily overrideable.

**Reply by kgilmer** [reply]
That works, also the lascaille look does not have special characters to denote workspaces.  Maybe useful if you'd like to integrate this into a look:

https://github.com/regolith-linux/regolith-look-extra/blob/main/usr/share/regolith-look/lascaille/i3-wm#L61
